### PR TITLE
Companion commit to PR #282

### PR DESCRIPTION
**Pull request by miri64**

### CODE REVIEW

File: sixlowpan/Makefile [hunk 1]
```diff
@@ -44,7 +44,7 @@ else ifeq ($(BOARD),native)
 	USEMODULE += nativenet
 endif
 
-export INCLUDES=-I$(RIOTBOARD)/$(BOARD)/include -I$(RIOTBASE)/core/include -I$(RIOTCPU)/$(CPU)/include -I$(RIOTBASE)/sys/lib -I$(RIOTBASE)/sys/include/ -I$(RIOTBASE)/drivers/include/ -I$(RIOTBASE)/drivers/cc110x_ng/include/ -I$(RIOTBASE)/sys/net/sixlowpan/include/
+export INCLUDES=-I$(RIOTBOARD)/$(BOARD)/include -I$(RIOTBASE)/core/include -I$(RIOTCPU)/$(CPU)/include -I$(RIOTBASE)/sys/lib -I$(RIOTBASE)/sys/include/ -I$(RIOTBASE)/drivers/include/ -I$(RIOTBASE)/drivers/cc110x_ng/include/ -I$(RIOTBASE)/sys/net/include/
```
Comment (OlegHahm) [inline sixlowpan/Makefile:48]: `-I$(RIOTBASE)/sys/net/destiny` is missing (for `msg_help.h`).

Comment (miri64) [inline sixlowpan/Makefile:48]: Yes, but we should change that in the future ;-)

Comment (OlegHahm) [inline sixlowpan/Makefile:48]: Ooops, I think I commented on the wrong Makefile. I meant tlayer:

```
diff --git tlayer/Makefile tlayer/Makefile
index 0b0e32b..5902fac 100644
--- tlayer/Makefile
+++ tlayer/Makefile
@@ -46,7 +46,7 @@ USEMODULE += uart0
 USEMODULE += posix
 USEMODULE += ltc4150

-export INCLUDES += -I$(RIOTBASE)/sys/include -I$(RIOTBASE)/drivers/include -I$(RIOTBASE)/drivers/cc110x_ng/include -I$(RIOTBASE)/sys/net/include
+export INCLUDES += -I$(RIOTBASE)/sys/include -I$(RIOTBASE)/drivers/include -I$(RIOTBASE)/drivers/cc110x_ng/include -I$(RIOTBASE)/sys/net/include -I$(RIOTBASE)/sys/net/destiny

 include $(RIOTBASE)/Makefile.include
 endif
diff --git tlayer/tlayer_shell.c tlayer/tlayer_shell.c
index 341b33c..d59943e 100644
--- tlayer/tlayer_shell.c
+++ tlayer/tlayer_shell.c
@@ -14,7 +14,7 @@
 #include "destiny/socket.h"

 #include "net_help.h"
-#include "../msg_help.h"
+#include "msg_help.h"

 #include "tlayer.h"
```

Comment (miri64) [inline sixlowpan/Makefile:48]: see 94bd279

**Comment by OlegHahm** [comment]
ACK and go.

**Pull request merged**